tools/report-converter/codechecker_report_converter/cli.py

```diff
@@ -50,6 +50,8 @@
     GolintAnalyzerResult  # noqa
 from codechecker_report_converter.pyflakes.analyzer_result import \
     PyflakesAnalyzerResult  # noqa
+from codechecker_report_converter.coccinelle.analyzer_result import \
```

Comment:
The `__init__.py` file is missing from the coccinelle directory so this tool will fail at this point.

You can copy and add this file from https://github.com/Ericsson/codechecker/blob/master/tools/report-converter/codechecker_report_converter/cppcheck/__init__.py